packages/wrangler/src/d1/create.ts

```diff
@@ -82,15 +88,16 @@ export const d1CreateCommand = createCommand({
 			}`
 		);
 		logger.log("Created your new D1 database.\n");
-		logger.log(
-			formatConfigSnippet(
-				{
-					d1_databases: [
-						{ binding: "DB", database_name: db.name, database_id: db.uuid },
-					],
-				},
-				config.configPath
-			)
+
+		// Handle binding name and config update using unified utility
```

Comment:
I love to see code comments and I must say you have done a great job in this PR!
But (nit) this one does not look to add value over the fn name?

```suggestion
```